leader/leader_test.go

```diff
@@ -0,0 +1,187 @@
+package leader
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("Leader", func() {
+
```

Comment:
I think this `Describe` layer is unnecessary. Every test in this suite (i.e. package) is a leader test.

Reply:
If I remove it, it doesn't seem to work. I renamed it to `Leader election` to better describe it. I would say, it works so I'd rather not change this.